application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -185,9 +183,10 @@ private MessageCreateAction createModMessage(String reportReason,
 
         MessageCreateAction message =
                 modMailAuditLog.sendMessageEmbeds(reportedMessageEmbed, reportReasonEmbed)
-                    .addActionRow(DiscordClientAction.Channels.GUILD_CHANNEL_MESSAGE.asLinkButton(
-                            "Go to Message", guild.getId(), reportedMessage.channelID,
-                            reportedMessage.id));
+                    .addActionRow(Button.link(
+                            "https://discord.com/channels/%s/%s/%s".formatted(guild.getId(),
```

Comment:
dont hardcode magic links in the middle of the code like that. thats bad, fragile and doesnt scale at all. if anything, at least create a proper helper method in a helper class. For example somewhere at `DiscordUtil` or whatever the class was called.

Reply:
done